### PR TITLE
Preview 9

### PR DESCRIPTION
**Pull request by ajhunyady**
Update main

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
             "dataflows-inline/word-counter",
             "dataflows-inline/word-probe",
           ]
-          ssdk-version: [ssdk-preview7]
+          sdf-version: [sdf-preview8]
```
Comment (crajcan) [inline .github/workflows/ci.yml]: 9?

Comment (ajhunyady) [reply .github/workflows/ci.yml]: fixed

**Comment by morenol** [comment]
It looks like there are conflicts with `main`

**Comment by morenol** [comment]
Should we merge #2 first?

**Comment by morenol** [comment]
I can solve all the conflicts if you want

**Comment by ajhunyady** [comment]
That's what @crajcan suggested as well. Let's do that.

**Comment by ajhunyady** [comment]
@crajcan @morenol please take over from here and do whatever is needed.

**Review by sehz — APPROVED** [review]
LGTM

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -28,14 +27,12 @@ jobs:
             "dataflows-inline/openai-callout",
             "dataflows-inline/word-counter",
             "dataflows-inline/word-probe",
-            "dataflows-composed/mask-user-info",
```
Comment (crajcan) [inline .github/workflows/ci.yml:31]: I think we want to keep this

File: dataflows-composed/split-sentence/Makefile [hunk 3]
```diff
@@ -1,6 +0,0 @@
-BASE_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST)))/../../)
-include $(BASE_DIR)/Makefile
-
-e2e:
```
Comment (crajcan) [inline dataflows-composed/split-sentence/Makefile:4]: Looks like this is removing the things I added in #2 to get the examples in `dataflows-composed/` to run on CI

File: README.MD [hunk 4]
```diff
@@ -13,9 +13,11 @@ rustup update
 
 #### Install Fluvio & SDF
 
+SDF if the abreviation for Stateful Dataflows.
```
Comment (crajcan) [inline README.MD]: is

File: .github/workflows/ci.yml (hunk 2 above)
Comment (morenol) [reply .github/workflows/ci.yml:31]: added again

**Review by crajcan — APPROVED** [review]
LGTM

**Pull request merged**